Kubernetes/omsagent.yaml

```diff
@@ -315,7 +315,7 @@ spec:
     dsName: "omsagent-ds"
    annotations:
     agentVersion: "1.10.0.1"
-    dockerProviderVersion: "6.0.0-0"
```

Comment:
Do you need these? They are already updated in your previous PR right?

Reply:
that was for ci_feature